### ISSUE TITLE
Add walkthrough / explanation of introductory GIF?

### ISSUE THREAD

**Issue by jsvine** [saulpw/visidata#54]
The analysis/workflow in the [introductory GIF](https://visidata.readthedocs.io/en/stable/img/screenshot.gif) is really neat, but a bit hard to follow.

New users might benefit from a step-by-step walkthrough of what's going on.

**Comment by saulpw** [comment]
I plan to create a new screenshot.gif this week, so this is very timely.  Do you think a separate bulleted walkthrough is best, or should I annotate the .gif itself?  Alternately I could link to a youtube video in which I talk about each of the commands that are being executed.

**Comment by saulpw** [comment]
@jsvine what do you think about this? https://github.com/saulpw/visidata/blob/stable/docs/tours.rst

**Comment by jsvine** [comment]
Fantastic. Very much what I had in mind.
